src/Database/PostgreSQL/PQTypes/Model/Trigger.hs

```diff
@@ -1,14 +1,17 @@
 -- |
 -- Module: Database.PostgreSQL.PQTypes.Model.Trigger
 --
--- Trigger name must be unique among triggers of same table. Only @CONSTRAINT@ triggers are
--- supported. They can only be run @AFTER@ an event. The associated functions are always
--- created with no arguments and always @RETURN TRIGGER@.
+-- Trigger name must be unique among triggers of same table.
+-- The associated functions are always created with no arguments
```

Comment:
I don't understand this sentence. I know it's been there before, but maybe let's remove it? Or if you know what it means, reword it?

Reply:
I removed this comment and wrote something similar inside the `Trigger` datatype.